src/stories/TextInput.stories.js

```diff
@@ -47,6 +47,7 @@ ErrorStory.args = {
 const ForceActiveLabel = Template.bind({});
 ForceActiveLabel.args = {
     label: 'Forced active label',
```

Comment:
I know you didn't add this but `Force active label` not `Forced`

Reply:
Also `ErrorStory` should be `Error`, with `name: 'InputWithError',` also adjusted to just `Error` and `errorText` as `Oops! Looks like there\'s an error.`